src/containers/my-resources/categories-container/CategoriesContainer.tsx

```diff
@@ -89,6 +131,18 @@ const CategoriesContainer = () => {
     fetchOnMount: false
   })
 
+  const onAdd = () => {
+    openModal({
+      component: (
+        <AddCategoriesModal
+          closeModal={closeModal}
+          createCategories={(categoryName?: CreateCategoriesParams) =>
```

Comment:
Can you just pass handleCreateCategory?
createCategories(handleCreateCategory)

and then, inside AddCategoriesModal pass category name to this callback

Reply:
> Can you just pass handleCreateCategory? createCategories(handleCreateCategory)
> 
> and then, inside AddCategoriesModal pass category name to this callback

Whoops, i forgot to remove this from here. It was already in AddCategoriesModal. Thanks 😅